rtv/__main__.py

```diff
@@ -37,15 +36,20 @@ def load_config():
     # read only the first existing config file
     for config_path in config_paths:
         if os.path.exists(config_path):
-            config.read(config_path)
+            parser.read(config_path)
             break
 
     defaults = {}
-    if config.has_section('rtv'):
-        defaults = dict(config.items('rtv'))
+    if parser.has_section('rtv'):
+        defaults = dict(parser.items('rtv'))
 
     if 'unicode' in defaults:
-        defaults['unicode'] = config.getboolean('rtv', 'unicode')
+        defaults['unicode'] = parser.getboolean('rtv', 'unicode')
+
+    if 'theme' in defaults:
+        config.theme = defaults['theme']
+    else:
+        config.theme = None
```

Comment:
FYI dictionaries have a `.get` method that does exactly what you're trying to do.

```python
config.theme = defaults.get('theme', None)
```
or simply
```python
config.theme = defaults.get('theme')
```